feat: add Fedora 38 build, build one hour behind the base image

castrojo opened the pull request:
Feel free to merge on approval!

castrojo committed:
feat: add Fedora 38 build, build one hour behind the base image

AdamIsrael commented:
The only problem I see is that the 38 build fails because an upstream repo doesn't exist yet: `https://copr.fedorainfracloud.org/coprs/kylegospo/gnome-vrr/repo/fedora-38/kylegospo-gnome-vrr-fedora-38.repo`

castrojo commented:
Yeah all the COPRs aren't making 38 builds yet, but I didn't want to edit the lines until we figure out #34 so we'll block this on that.

castrojo committed:
chore: drop 38 builds

38 will be busted for a bit for this image, so let's just make the cron adjustments for now.
